[feat] add `docs` command

arrayappy opened the pull request:
#### Problem #29

#### Summary of Changes
- [x] tested the feature of opening websites on mac, windows, and linux
- [x] updated index.ts to include `docs` command
- [x] updated README.md to include docs section
- [x] ensured prettier formatting

Fixes #29

nickfrosty commented on src/commands/docs.ts:
```diff
@@ -0,0 +1,140 @@
+        titleMessage("Open documentation");
+      }
+
+      shellExecInSession({
+        command: `${openCommand} ${DOCS_INFO[tool].url}`,
+        outputOnly: options.outputOnly,
+        args: [],
+      });
```
instead of exec in session, which adds some unneeded output for the user, we can use shellExec:
```suggestion
      shellExec(`${openCommand} ${DOCS_INFO[tool].url}`);
```

nickfrosty commented on src/commands/docs.ts:
```diff
@@ -0,0 +1,140 @@
+  $ mucho docs solana  # open Solana documentation`,
+    )
+    .action(async (tool, options) => {
+      if (!options.outputOnly) {
+        titleMessage("Open documentation");
+      }
```
since output only does not really do anything here, we can skip this `if`:
```suggestion
    titleMessage("Open documentation");
```

nickfrosty commented on src/commands/docs.ts:
```diff
@@ -0,0 +1,140 @@
+        .choices(TOOLS)
+        .default("mucho"),
+    )
+    .addOption(COMMON_OPTIONS.outputOnly)
```
output only does not serve a purpose here, so lets just remove it
```suggestion
```

arrayappy replied:
resolved all these changes with the below commit

arrayappy replied:
awesome. resolved all these changes and added a commit.

nickfrosty reviewed (changes requested):
this is really great work, i suggested a few changes to help round this all out.

on top of the suggested changes, can you also print the selected docs link itself? this way when the user opens the url, they will still see it in the console. as opposed to nothing being in the console

arrayappy commented:
awesome, i've added two commits

1. addressed all the pr review changes
2. resolved conflicts with master branch on readme and index.ts

arrayappy commented:
awesome, i've added three commits

1. addressed all the pr review changes
2. resolved conflicts with master branch on readme and index.ts
3. changeset

nickfrosty reviewed (approved):
Thanks for this PR and changes! You are now the first external contributor on mucho :)
